src/App.js

```diff
@@ -23,8 +23,9 @@ function App() {
   }, []);
 
   const newPet = () => {
-    randomPet(setPetName);
-    setHunger(100);
+    randomPet()
+      .then(setPetName)
+      .then(setHunger(100));
```

Comment:
@starsuit this is going to be immediately invoked rather than waiting for the promise to resolve. You probably want to group these into a single `.then()`

```js
randomPet()
  .then(name => {
    setName(name);
    setHunger(100);
  })
```